data/generateData.js

```diff
@@ -177,8 +181,11 @@ function getTopRoundsByScore(roundsByScore) {
   return topRounds;
 }
 
-function getStats(rounds) {
-  const statsByPlayer = Object.values(rounds.reduce(getStatsByPlayer, {}));
+function getStats(rounds, maxRoundsPerPlayer) {
+  const boundStatsByPlayer = getStatsByPlayer.bind(null, maxRoundsPerPlayer);
```

Comment:
Let's not do this. It's super-confusing to change the standard signature for a reduce function.

Reply:
Okay, I’ll change it to a function that returns a function with the standard sig.